server/service/integration_core_test.go

```diff
@@ -3089,6 +3089,9 @@ func (s *integrationTestSuite) TestLabels() {
 	s.DoJSON("GET", fmt.Sprintf("/api/latest/fleet/labels/%d/hosts", lbl2.ID), nil, http.StatusOK, &listHostsResp)
 	assert.Len(t, listHostsResp.Hosts, len(hosts))
 
+	s.DoJSON("GET", fmt.Sprintf("/api/latest/fleet/labels/%d/hosts?order_key=%s&after=%d", lbl2.ID, "id", 1), nil, http.StatusOK, &listHostsResp)
+	assert.Len(t, listHostsResp.Hosts, 2)
```

Comment:
Same comment as https://github.com/fleetdm/fleet/pull/14099/files#r1334820097, we may want to check contents of `listHostsResp`

Reply:
Done!